content/backmatter.tex

```diff
@@ -522,6 +522,25 @@ \section{Version 1.5}
 \CONST{SHMEM\_MALLOC\_ATOMICS\_REMOTE} and \CONST{SHMEM\_MALLOC\_SIGNAL\_REMOTE}.
 \\ See Section \ref{subsec:shmmallochint} and \ref{subsec:library_constants}.
 %
+\item Changed the broadcast function to update the dest object on all PEs,
+including the root PE.
+\\ See Section \ref{subsec:shmem_broadcast}.
+%
+\item Clarified error codes returned from collectives.
+\\ See Section \ref{subsec:coll}.
```

Comment:
I believe that the active-set based collectives did not return error codes. Since this only applies to the new collectives, it seems unnecessary to note this as a separate changelog entry.

Reply:
Will delete the change log. However, the main text does not specify that the error codes are team-based collectives specific. How about we change the title?
```
Error codes returned from collectives
```
->
```
Error codes returned from team-based collectives
```

Reply:
Did we update the legacy collectives to also return error codes? I think this was discussed but ultimately not incorporated.

Reply:
We didn't. Same for `shmem_barrier_all` and `shmem_sync_all`.  Do you think we should leave the paragraph title as is?

Reply:
A clarification is something we would make to existing text. Given that the error codes are newly added in OpenSHMEM 1.5, this changelog entry seems confusing to me. I think it would be fine to remove it and assume that this change is covered by another entry indicating the new team-based collectives. However, if you feel like it's helpful to call out this change in particular, it could be phrased a bit more like a new addition:

> Specified that \openshmem defines only success and failure error codes for team-based collective routines.

Reply:
Sorry for my late reply. I agree that the error code change is covered by the new team-based collectives entry. Thus, I deleted the change log entry and updated the paragraph title to: "Error codes returned from team-based collectives"